lib/src/usecase.dart

```diff
@@ -128,7 +129,8 @@ abstract class UseCase<T, Params> {
 
   /// Adds a [StreamSubscription] i.e. the subscription to the
   /// [Observable] to the [CompositeSubscription] list of subscriptions.
-  void _addSubscription(StreamSubscription subscription) {
+  @protected
```

Comment:
Why you change this method into protected instead of keep it private?

Reply:
Going to change it back. I was going to use it in BackgroundUseCase but it is no longer needed. I'm going to change it back to private once I push more code.

Reply:
Great. Also, if it's possible, please add the tag [WIP] before the PR title, so it gets more clear that is an in progress PR.

I will come back and review the changes once the PR is done!